setup.py

```diff
@@ -410,6 +464,20 @@ def get_hdf5_version(headername):
         # Don't delete this argument. It maybe useful for distutils
         # when adding more flags later on
         # sys.argv.remove(arg)
+    elif arg.find('--use-pkgconfig') == 0:
+        if arg == '--use-pkgconfig':
+            USE_PKGCONFIG = True
+        else:
+            USE_PKGCONFIG = arg.split('=')[1]
+
+
+if isinstance(USE_PKGCONFIG, str):
+    if USE_PKGCONFIG.upper() in ('TRUE', 'YES', '1', 'ON', 'OK'):
```

Comment:
do we really need all these variants? what about 'SI', 'OUI', 'PLEASE', 'ENABLED' ? :)